odoo/addons/test_http/tests/test_http.py

```diff
@@ -380,8 +380,7 @@ def test_models3_stargate_ko(self):
 class TestHttpMisc(TestHttpBase):
     def test_misc0_redirect(self):
         res = self.nodb_url_open('/test_http//greeting')
-        self.assertEqual(res.status_code, 301)
-        self.assertEqual(urlparse(res.headers.get('Location', '')).path, '/test_http/greeting')
+        self.assertEqual(res.status_code, 404)
```

Comment:
For futur generations, this test is still valid because we use `nodb_url_open` so http_routing doesn't kicks in and redirect us.